electron-app/src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app id="rotki">
-    <div v-if="logged">
+    <div v-show="logged">
```

Comment:
was the flicker because we were removing the element from dom?

Reply:
the If is something I added in the first commit, the flicker is caused by the fact that the full app is actually rendering (and visible) before the overlay is, this change makes sure it's hidden before the welcome screen overlay comes up.